hw1/hw1_mzw7af_vl4kz.py

```diff
@@ -107,22 +107,25 @@ def why(expr):
 def main():
     for line in sys.stdin:
         argArray = line.strip().split(" ")
-        
-        if argArray[0].lower() == "teach":
+        command = argArray[0].lower()
+
+        if command == "teach":
             if argArray[3] == "=":
-                teachVar(argArray[1], argArray[2], line.strip().split(" = ")[1])
+                ruleDef = line.strip().split(" = ")[1]
+                teachVar(argArray[1], argArray[2], ruleDef.replace('"', ""))
```

Comment:
weird way of saying a quotation mark lmao. You can do an escape character "\"" or something like that, but yolo this works too